pkg/controllers/reconciler.go

```diff
@@ -17,24 +17,32 @@ package controllers
 
 import (
 	"context"
+	"path/filepath"
+	"strings"
 	"sync"
 
 	"github.com/go-logr/logr"
+	"github.com/google/go-cmp/cmp"
 	apps "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/diff"
+	"k8s.io/apimachinery/pkg/util/version"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
 var (
-	bKeeper = &bookKeeper{}
+	UpgradePlugins bool
```

Comment:
If we set a default value here, the test cases do not need to import and set the value

Reply:
@mythi 
In principle, not a bad idea, I just tried, but there's a problem to create initial 0.22.0 CRs in envtests, they are upgraded immediately. I.e. in one way or another this should be controlled in envtests.